packages/instant/src/containers/connected_account_payment_method.ts

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+
+import { PaymentMethod } from '../components/payment_method';
+import { Action, actions } from '../redux/actions';
+import { State } from '../redux/reducer';
+import { Account, Network, StandardSlidingPanelContent } from '../types';
+
+export interface ConnectedAccountPaymentMethodProps {}
+
+interface ConnectedState {
+    account: Account;
+    network: Network;
+}
+
+interface ConnectedDispatch {
+    openStandardSlidingPanel: (content: StandardSlidingPanelContent) => void;
```

Comment:
Want to start using the pattern of `Pick`-ing `openStandardSlidingPanel` from `PaymentMethod` here?